src/txs/Tx.tsx

```diff
@@ -335,7 +335,7 @@ function Tx<TxValues>(props: Props<TxValues>) {
                 <Read amount={balance} token={token} decimals={decimals} />
               </dd>
 
-              <dt>{t("Balance after tx")}</dt>
+              <dt>{t("Balance after transaction")}</dt>
```

Comment:
I kept this as it is, because it takes a bit too much space, and we would have to update the translations too